src/components/consul/detail/ConsulCommentListItem.js

```diff
@@ -0,0 +1,219 @@
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import { View, StyleSheet } from 'react-native';
+import { useForm } from 'react-hook-form';
+import { useMutation } from 'react-apollo';
+
+import { momentFormatUtcToLocal } from '../../../helpers';
+import { RegularText } from '../../Text';
+import { WrapperRow, WrapperVertical } from '../../Wrapper';
+import { Button } from '../../Button';
+import { Touchable } from '../../Touchable';
+import { colors, normalize, texts, Icon } from '../../../config';
+import { Input } from '../Form';
+import { ConsulClient } from '../../../ConsulClient';
+import { ADD_REPLY_TO_COMMENT, CAST_VOTE_ON_COMMENT } from '../../../queries/Consul';
+
+const text = texts.consul;
```

Comment:
why have you created this text? what is you benefit here?

Reply:
It is a method made to shorten the code. The reason I do this is because I only use the text files of the consul.